services/motion/motion_test.go

```diff
@@ -1544,3 +1545,182 @@ func validMoveOnGlobeRequest() MoveOnGlobeReq {
 		Extra: nil,
 	}
 }
+
+func TestMoveOnMapReq(t *testing.T) {
```

Comment:
Super minor but can we move this up next to where the test for MoveOnGlobeReq is?

Reply:
you got it!